clerk/tokens.go

```diff
@@ -0,0 +1,100 @@
+package clerk
+
+import (
+	"fmt"
+	"time"
+
+	"gopkg.in/square/go-jose.v2"
+
+	"gopkg.in/square/go-jose.v2/jwt"
+)
+
+type Claims struct {
+	jwt.Claims
+	UserClaims
+}
+
+type UserClaims struct {
+	Name                string `json:"name"`
+	Picture             string `json:"picture"`
+	UpdatedAt           int64  `json:"updated_at"`
+	GivenName           string `json:"given_name,omitempty"`
+	FamilyName          string `json:"family_name,omitempty"`
+	PreferredUsername   string `json:"preferred_username,omitempty"`
+	Gender              string `json:"gender,omitempty"`
+	Birthdate           string `json:"birthdate,omitempty"`
+	Email               string `json:"email,omitempty"`
+	EmailVerified       bool   `json:"email_verified,omitempty"`
+	PhoneNumber         string `json:"phone_number,omitempty"`
+	PhoneNumberVerified bool   `json:"phone_number_verified,omitempty"`
+}
+
+// DecodeToken decodes the session jwt token.
```

Comment:
:upside_down_face: Perhaps we can add something in the godoc about the token not being verified, just decoded?